pkg/apiserver/openapi/zz_generated.openapi.go

```diff
@@ -443,6 +443,13 @@ func schema_pkg_apis_clusterinformation_v1beta1_AntreaAgentInfo(ref common.Refer
 							},
 						},
 					},
+					"apiPort": {
+						SchemaProps: spec.SchemaProps{
+							Description: "Agent condition contains types like AgentHealthy",
```

Comment:
Not introduced by this PR, but it looks like all OpenAPI description are wrong.
Maybe related to how we write comments.

Reply:
Yeah, it looks like the generator would use the nearest above comment of the field as the comment. I think we should fix this issue in another PR?